xbmc/application/ApplicationPlayer.h

```diff
@@ -55,6 +55,8 @@ class CApplicationPlayer : public IApplicationComponent
   void FlushRenderer();
   void SetRenderViewMode(int mode, float zoom, float par, float shift, bool stretch);
   float GetRenderAspectRatio() const;
+  bool GetRects(CRect& source, CRect& dest, CRect& view);
+  unsigned int GetOrientation();
```

Comment:
Any reason these two methods not being `const`?

Reply:
no, there was a little bit of trickery I missed. done.